src/cli/repo/s3.go

```diff
@@ -193,7 +194,7 @@ func connectS3Cmd(cmd *cobra.Command, args []string) error {
 
 	repoID := cfg.RepoID
 	if len(repoID) == 0 {
-		repoID = "not_found"
+		repoID = events.RepoID
```

Comment:
This is an opaque application of a const, since it reads like you're providing a valid RepoID, not the `"not_found"` string.  A clearer implementation would be to add a const like `RepoIDNotFound`, and use that in these locations instead.  This pattern would also make it easier to understand checks like on line 232 in repository.go, which is currently confusing.